api/app/jobs/common_model_fetchers.py

```diff
@@ -373,9 +433,10 @@ def _process_model_run_for_station(self,
                 noon_prediction = construct_interpolated_noon_prediction(
```

Comment:
Was this part of the code was meant to handle interpolating the noon predictions?

Reply:
Yes, it constructs the 'prediction' part for GFS and GDPS properly as far as I can tell. Then I had to tweak the part for constructing the bias corrected values.